api/sql/schema/750$rule.decision.fetch.sql

```diff
@@ -112,64 +121,204 @@ BEGIN
         @amountWeekly = ISNULL(c.amountWeekly, 0),
         @countWeekly = ISNULL(c.countWeekly, 0),
         @amountMonthly = ISNULL(c.amountMonthly, 0),
-        @countMonthly = ISNULL(c.countMonthly, 0)
+        @countMonthly = ISNULL(c.countMonthly, 0),
+        @limitCredentials = l.[credentials],
+        @limitMaxAmount = l.maxAmount
     FROM
         @matches AS c
     JOIN
         [rule].limit AS l ON l.conditionId = c.conditionId
     WHERE
-        l.currency = @currency
+        l.currency = @currency 
+    AND 
+        (
+             @amount < l.minAmount OR
+             @amount > ISNULL(@maxAmountParam, l.maxAmount) OR
+             @amount + @amountDaily > l.maxAmountDaily OR
+             @amount + @amountWeekly > l.maxAmountWeekly OR
+             @amount + @amountMonthly > l.maxAmountMonthly OR
+             @countDaily >= l.maxCountDaily OR
+             @countWeekly >= l.maxCountWeekly OR
+             @countMonthly >= l.maxCountMonthly        
+        )   
```

Comment:
`AND @credentials & ISNULL(@credentialsCheck, l.credentials) = ISNULL(@credentialsCheck, l.credentials)`